src/org/exist/indexing/StreamListener.java

```diff
@@ -43,6 +43,13 @@
         STORE,
 
         /**
+         * Mode for replacing the nodes of a document with another document
+         * this is really a group mode, which is later followed by {@link #REMOVE_ALL_NODES}
+         * and then {@link #STORE}
+         */
+        REPLACE_DOCUMENT,
```

Comment:
that call is [here](https://github.com/adamretter/exist/blob/c841f6b7d17308be143b1e218a40750e0697264d/src/org/exist/collections/MutableCollection.java#L1261)